config/__init__.py

```diff
@@ -172,6 +172,7 @@ def update_status_config(milestones_content):
 # labels that we allow to be added via a `label` GET param, when
 # creating an issue.
 EXTRA_LABELS = [
+    'browser-focus-geckoview',
```

Comment:
this adds an extra label of the browser-* category

Will it work? with the new webhook logic. Maybe this requires a test. because there will be a `browser-focus-geckoview` coming from extra_labels AND a `browser-firefox-mobile` coming from browser_label.

Will this end up with a `browser-fixme`?

Reply:
...good question. let's go find out.